app/scripts/modules/kubernetes/serverGroup/configure/CommandBuilder.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.kubernetes.serverGroupCommandBuilder.service', [
+  require('../../../core/cache/deckCacheFactory.js'),
```

Comment:
Replace this line with:
```javascript
require('../../../core/config/settings.js'),
```
I know you were just copying existing code, but that code was wrong! So we might as well fix it here.